handbook/engineering/index.md

```diff
@@ -15,6 +15,8 @@
 - [Deployments](deployments.md)
 - [On-call](on_call/index.md)
 - [Prometheus](prometheus.md)
+- [Security](security.md)
```

Comment:
Why is this part of the diff?
```suggestion
```

Reply:
I thought I had accidentally deleted it but someone else must have taken it out on a different branch concurrently. Will remove